config/groovy/util.groovy

```diff
@@ -117,8 +117,14 @@ switch (cleanerArgs[0]) {
     case "update-all":
         println "We're updating every $itemType"
         println "List of local entries: ${common.retrieveLocalItems()}"
-        for (item in common.retrieveLocalItems()) {
-            common.updateItem(item)
+        if (cleanerArgs.contains("--continue")) {
```

Comment:
The original reason I chose `--continue` is it lines up with some git commands so I thought it would be easier to remember. For example, If a git merge fails due to conflicts, you resolve the conflicts and call `git merge --continue`. I also did the two dashes convention for full words (-u is short for --set-upstream). 

I'm going change this to `-skip-recently-updated` to closer match the other flags listed in the printUsage.